frontend/src/common/queries/cellCards.ts

```diff
@@ -96,6 +96,11 @@ export interface CellOntologyTreeResponse {
   _children?: this[];
 }
 
+export const useCellOntologyTree =
+  (): UseQueryResult<CellOntologyTreeResponse> => {
+    return useCellCardQuery<CellOntologyTreeResponse>(TYPES.CELL_ONTOLOGY_TREE);
```

Comment:
added wrapper hooks to for convenience and am using generics to return the correct response type